modules/govuk_jenkins/templates/jobs/build_fpm_package.yaml.erb

```diff
@@ -0,0 +1,26 @@
+---
+- scm:
+    name: build_fpm_package
```

Comment:
In other jobs the scm name and job name are different, I'm not sure if this is a requirement or just an oddity?